code/src/components/MovieDetails.js

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from 'react'
+import { useParams, useHistory } from 'react-router-dom'
+
+import './movieDetails.css'
+
+import { DETAILS_URL } from '../utils/urls'
+import { Loading } from '../components/Loading'
+
+
+export const MovieDetails = () => {
+
+	const { movieId } = useParams();
+	const history = useHistory(); 
+
+  	const [details, setDetails] = useState([]);
+	const [loading, setLoading] = useState(true);
+
+	const detailStyling = {
+		backgroundImage: `linear-gradient(rgba(0, 0, 0, 0) -70%, rgb(0, 0, 0) 100%), url(https://image.tmdb.org/t/p/w1280/${details.backdrop_path})`,
```

Comment:
Good that you tried out styled components. But remember to do all or nothing, be consistent.

Reply:
It is not a styled component, Maria found a solution to put all of the styling in an object instead :)